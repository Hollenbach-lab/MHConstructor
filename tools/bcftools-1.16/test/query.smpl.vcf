##fileformat=VCFv4.2
##contig=<ID=chr1>
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	00	11
chr1	10000	.	A	C	.	.	.	GT	0/0	1/1
