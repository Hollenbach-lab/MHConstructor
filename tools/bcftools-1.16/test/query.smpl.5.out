11 1/1
00 0/0
